src/neuroconv/tools/data_transfers/_aws.py

```diff
@@ -30,3 +36,283 @@ def estimate_s3_conversion_cost(
     cost_gb_m = 0.08 / 1e3  # $0.08 / GB Month
     cost_mb_s = cost_gb_m / (1e3 * 2.628e6)  # assuming 30 day month; unsure how amazon weights shorter months?
     return cost_mb_s * total_mb_s
+
+
+def submit_aws_batch_job(
+    *,
+    job_name: str,
+    docker_image: str,
+    command: Optional[str] = None,
+    job_dependencies: Optional[List[Dict[str, str]]] = None,
+    region: str = "us-east-2",
+    status_tracker_table_name: str = "neuroconv_batch_status_tracker",
+    iam_role_name: str = "neuroconv_batch_role",
+    compute_environment_name: str = "neuroconv_batch_environment",
+    job_queue_name: str = "neuroconv_batch_queue",
+    job_definition_name: Optional[str] = None,
+    minimum_worker_ram_in_gb: float = 4.0,
+    minimum_worker_cpus: int = 4,
+    submission_id: Optional[str] = None,
+) -> Dict[str, str]:
+    """
+    Submit a job to AWS Batch for processing.
+
+    Parameters
+    ----------
+    job_name : str
+        The name of the job to submit.
+    docker_image : str
+        The name of the Docker image to use for the job.
+    command : str, optional
+        The command to run in the Docker container.
+        Current syntax only supports a single line; consecutive actions should be chained with the '&&' operator.
+    job_dependencies : list of dict
```

Comment:
The only thing I'm cautious of here is offering too much information about the CURRENT `boto3` API given how much it can change even over the course of a year

My approach here is just to expose the ability to pass the arguments; perhaps we can simply link out to `boto3` documentation and tell users to reference that for exact syntax?

Most of the specifics for our own usage though will be handled in the following PR(s)

Reply:
Yeah, a link would be fine. Or just "refer for "x" on the boto3 documentation to see more" if we don't want to deal with possible dead links.